### ISSUE TITLE
Tools And Languages

### ISSUE THREAD

**Issue by roshanlam** [geohot/hammer-website#5]
What tools and programming languages is going to be used? Because I want to recommend NodeJS and C/C++, you use them both using a library called ffi. It's pretty cool and we (the developers/programmers) could add some cool AI tech that will might make it a competition against search engines like google and duck duck go. We could also make the website's UI look awesome by using ReactJS, VueJS or some other framework/tool. 
 
https://github.com/node-ffi/node-ffi
https://nodejs.org/api/addons.html

**Comment by 0xffset** [comment]
@roshanlam, I agree with you. The use of Machine Learning Algorithms might improve the searching result, but. remember that the Google Search Engine works with the Ranking Page Algorithm. thus, can implement this algorithm with some AI algo too.
